main.tf

```diff
@@ -1,15 +1,34 @@
+locals {
+  values = yamlencode({
+    "awsRegion" : data.aws_region.current.name,
+    "autoDiscovery" : {
+      "clusterName" : var.cluster_name
```

Comment:
I don't think this values are for ingress-nginx helm chart, see possible values [here](https://github.com/kubernetes/ingress-nginx/tree/main/charts/ingress-nginx).